app/helpers/versions_helper.rb

```diff
@@ -2,35 +2,80 @@ module VersionsHelper
   ##
   # Groups functions related to change description
   ##
-  def link_if_alive(version, link_text, link_url)
-    version.item ? link_to(link_text, link_url) : link_text
+  def link_if_alive(version, link_text, link_url, conference)
+    version.item && conference ? link_to(link_text, link_url) : "#{link_text} with ID #{version.item_id}"
+  end
+
+  def link_to_conference(conference_id)
+    return 'deleted conference' if conference_id.nil?
+
+    conference = Conference.find_by(id: conference_id)
+    if conference
+      link_to conference.short_title,
+              edit_admin_conference_path(conference.short_title)
+    else
+      short_title = current_or_last_object_state('Conference', conference_id).try(:short_title) || ''
+      " #{short_title} with ID #{conference_id}"
```

Comment:
My reasoning for introducing this is that when we look at the changelog it is very possible that we want to check who did what (possibly something they shouldn't have done). If the record is deleted, or if things got deleted in a way they shouldn't have, we won't have proper access to the information we want. An ID might prove helpful to allow for organizers to further investigate what happened manually (the conference_id will allow us to manually search the versions in our DB for the item of this particular conference. If the conference record is deleted, we cannot get that information only with its short_title).